docs/en_US/Compression/v2_pruning_algo.rst

```diff
@@ -478,6 +702,29 @@ User configuration for Simulated Annealing Pruner
 
 .. autoclass:: nni.algorithms.compression.v2.pytorch.pruning.SimulatedAnnealingPruner
 
+Performance Test
+^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Model
+     - Accucacy
+     - Parameters
+     - FLOPs
+     - Sparsity
+   * - VGG-16
+     - 93.86%
+     - 14.98M
+     - 313.46M
+     - 
+   * - Pruned VGG-16
+     - 33.6% (wo FT)
+     - 
+     - 
```

Comment:
what is the meaning of "wo FT", and there are also two missing numbers

Reply:
At first I want the result 'without finetune', but this time it has been normal.
Just like the workflow: `prune -> speedup -> finetune`